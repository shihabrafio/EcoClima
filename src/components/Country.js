import React from 'react';

const Country = (props) => {
  const { name, value } = props;
  const parsedName = React.createElement('div', { dangerouslySetInnerHTML: { __html: name } });
  const style = { fontWeight: '700', color: '#efefef' };
  return (
    <div className="modal">
      {parsedName}
      <div style={style}>
        {value}
      </div>
    </div>
  );
};

export default Country;
