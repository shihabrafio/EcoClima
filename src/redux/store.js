import { configureStore } from '@reduxjs/toolkit';
import CountryReducer from './countrySlice';

const store = configureStore({
  reducer: CountryReducer,
});

export default store;
