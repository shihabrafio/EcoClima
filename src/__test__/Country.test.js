import React from 'react';
import { render as rtlRender } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Country from '../components/Country';

const render = (component) => rtlRender(
  <Provider store={store}>
    {component}
  </Provider>,
);

describe('Country', () => {
  it('render Country Components', () => {
    render(<Country />);
    expect(Country).toMatchSnapshot();
  });
});
