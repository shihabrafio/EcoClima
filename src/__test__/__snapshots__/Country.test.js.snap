// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Countires render Header Components 1`] = `[Function]`;

exports[`Country render Country Components 1`] = `[Function]`;
